site/themes/contour/assets/scss/var.scss

```diff
@@ -0,0 +1,11 @@
+$white: #ffffff;
```

Comment:
any reason this file as added? this looks like a duplicated of the _variables.scss file. can you remove it?

Reply:
It was by mistake and I've removed that change. I've also added the changelog file please look into it.